doc/troubleshooting/README.md

```diff
@@ -42,6 +42,19 @@ After the package is provisioned, the users need to log into their Windows accou
 
 ## Common Issues
 
+### Executing `winget` exits with no message
```

Comment:
I've also seen it described as "freezed" output, and some users have experienced the flickering cursor to never stop (https://github.com/microsoft/winget-cli/issues/3995). Maybe we can have multiple titles for users to quickly find this (not sure how good they'll look in markdown)

```
### Executing `winget` exits with no message
### winget commands hang indefinitely
```